client/src/components/MaterialsManager/MaterialsManager.jsx

```diff
@@ -381,9 +381,11 @@ const MaterialsManager = ({ projectID, show, onClose, ...props }) => {
 
   return (
     <Modal size="fullscreen" open={show} onClose={onClose} {...props}>
-      <Modal.Header>Manage Ancillary Materials</Modal.Header>
+      <Modal.Header>Manage Project Materials</Modal.Header>
```

Comment:
We probably shouldn't start shifting the UI for this until the backend implementation has been shifted towards Projects. Uploaded files will start to be listed in responses from the Books API.

Reply:
Removed. This was WIP that wasn't intended for this commit